tests/views/graph_manager_tests.py

```diff
@@ -189,8 +189,6 @@ def setUpTestData(cls):
         cls.GRAPH_ID = str(graph.pk)
         cls.NODE_COUNT = 5
 
-        cls.graph = graph
```

Comment:
Django cleans up the db for us, but not the model instances -- we had some tests publishing and unpublishing the same graph instance, which could introduce some test isolation issues, so I just removed this reference to make it harder to do. I think I was the one that added this (whoops)

Reply:
Lies. TIL django [cleans up](https://docs.djangoproject.com/en/5.1/topics/testing/tools/#django.test.TestCase.setUpTestData) the model instances for you. i'll revert these unnecessary changes.